binaries/cli/src/command/stop.rs

```
@@ -127,7 +131,9 @@ fn stop_dataflow_by_name(
         serde_json::from_slice(&reply_raw).wrap_err("failed to parse reply")?;
     match result {
         ControlRequestReply::DataflowStopped { uuid, result } => {
-            handle_dataflow_result(result, Some(uuid))
+            // For stop commands, we consider it successful if we successfully stopped the dataflow,
+            // even if the dataflow was already in a failed state
+            handle_dataflow_result_with_options(result, Some(uuid), true)
```

same as above (don't hide error)